### ISSUE TITLE
Error on first start: Unsupported Config Type "config"

### ISSUE THREAD

**Issue by kondr1** [zaquestion/lab#143]
My first try to use lab:
```
$ curl -s https://raw.githubusercontent.com/zaquestion/lab/master/install.sh | bash
[sudo] password for kondr:
Successfully installed lab into /usr/local/bin/
$ lab
Enter default GitLab host (default: https://gitlab.com):
Enter default GitLab user: kondr1
Create a token here: https://gitlab.com/profile/personal_access_tokens
Enter default GitLab token (scope: api): 2018/04/23 15:08:47 main.go:38: Unsupported Config Type "config"
```
I try make config manualy and it worked. But first run should not be with error.

**Comment by zaquestion** [comment]
Drat, thanks for reporting! Will ahbe it fixed right away.

**Comment by zaquestion** [comment]
@kondr1 Thanks again for reporting! This is fixed in `0.11.1`
